README.md

```diff
@@ -25,8 +25,6 @@ sudo apt-get install nodejs cmake build-essential swig python3.8
 sudo npm install --global npm
```

Comment:
This program outputs the variable twice. Not sure if it's intended behavior.

![image](https://user-images.githubusercontent.com/126523/222990008-88365ccd-2e9e-43c0-a104-d18acfffee1b.png)

And this program outputs it 4 times:

![image](https://user-images.githubusercontent.com/126523/222990068-090b64d1-da58-4873-b84c-1f8982546155.png)

Reply:
Show variable outputs the variable every second, to stop it from showing you can use hide variable. Seems to be the intended flow, though we could probably make changes if we choose to.